project/styles/components/content/facts-timetable-regular.scss

```
@@ -0,0 +1,34 @@
+.facts-timetable-regular {
+  padding: 12px 0;
+  margin: 24px 0;
+
+  @include breakpoint($bp-s) {
+    border-left: 2px solid $grey2;
+    margin: 24px;
+    padding: 12px 24px;
+  }
+}
+
+.fact-timetable-regular__title {
+  @include title3;
+  padding-bottom: 16px;
+}
+
+.fact-timetable-regular-item {
+  display: block;
+  color: $primary-color;
+
+  & ~ .fact-timetable-regular-item {
+    padding-top: 6px;
+  }
+
+  th {
+    font-family: $bold;
+    min-width: 100px;
```

ik ben niet zo gek op vaste breedtes voor tekstcontainers. Dat gaat met internationalisering vaak mis.

```suggestion
    padding: 0 16px 8px 0;
```